project_task_stage_change_restriction/tests/test_stage_change_restriction.py

```
@@ -0,0 +1,167 @@
+    def setUpClass(cls):
+        super().setUpClass()
+        env = cls.env
+
+        gp_user = env.ref("project.group_project_user")
+        gp_manager = env.ref("project.group_project_manager")
+        try:
+            cls.group_sales_admin = env.ref("sale.group_sale_manager")
+        except ValueError:
+            cls.group_sales_admin = env["res.groups"].create(
+                {
+                    "name": "Sales / Manager (Test)",
```

Ruff (0.11.9)

String contains ambiguous ` ` (NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF001)